tcf_core/settings.py

```diff
@@ -168,21 +169,26 @@
 if not DEBUG:
 
     # Heroku configuration.
-    import django_heroku
-    django_heroku.settings(locals())
+    # import django_heroku
+    # django_heroku.settings(locals())
 
     # Gather information from environment variables.
 
     HOSTNAME = os.environ.get('HOSTNAME', None)
     PUBLIC_IPV4 = os.environ.get('PUBLIC_IPV4', None)
 
-    ALLOWED_HOSTS = [
-        'tcf.brianyu.dev',
-        'thecourseforum.com',
-        'staging.thecourseforum.com',
-        'dev.thecourseforum.com',
-        'thecourseforum-staging.herokuapp.com',
-    ]
+    # SECURITY WARNING: App Engine's security features ensure that it is safe to
+    # have ALLOWED_HOSTS = ['*'] when the app is deployed. If you deploy a Django
+    # app not on App Engine, make sure to set an appropriate host here.
+    # See https://docs.djangoproject.com/en/1.10/ref/settings/ (from GCP documentation)
+    ALLOWED_HOSTS = ['*']
+
+    # ALLOWED_HOSTS = [
+    #     'thecourseforum.com',
+    #     'staging.thecourseforum.com',
+    #     'dev.thecourseforum.com',
+    #     'prod.thecourseforum.com',
+    # ]
```

Comment:
Delete please

Reply:
So, what are we doing with the current heroku staging and dev instances?

Reply:
I believe that those define HOSTNAME, so that'll be handled in line `194`

Reply:
Ah I see. Do I need to uncomment the django_heroku.settings(locals()) section to still be able to deploy to Heroku?